src/components/chart/range-annotations-options.js

```diff
@@ -24,7 +28,8 @@ class RangeAnnotationsOptions {
                     style: {
                         color: this.constants.labelColor,
                         fontSize: this.constants.defaultFontSize,
-                        width: rangeAnnotation.labelWidth ? rangeAnnotation.labelWidth : 150,
+                        // this property is ignored if rangeAnnotation.labelInside is undefined
```

Comment:
Should this read "this property is undefined if…"?

Reply:
No, it's ignored by Highcharts, and not set as an inline style, if undefined. I can make that clearer.